src/simplay-web/src/event/ContainerSetCapacityEvent.ts

```diff
@@ -12,6 +17,14 @@ export class ContainerSetCapacityEvent extends Event {
     super(forId, timestamp, EventAction.CONTAINER_SET_CAPACITY, args);
   }
   execute(context: SimplayContext) {
-    throw new Error('Method not implemented.');
+    const entityDisplayObject = getEntityDisplayObjectById(
```

Comment:
What happens if entity with this id does not exist or if the capacity param was not set?